### PR TITLE
[happyguestmx__web-utilities] Provide types for @happyguestmx/web-utilities

### PR DESCRIPTION
**Pull request by HappyGuest**
Provide types for [@happyguestmx/web-utilities](https://www.npmjs.com/package/@happyguestmx/web-utilities)

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/happyguestmx__web-utilities/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,313 @@
+// Type definitions for @happyguestmx/web-utilities 3.17
+// Project: https://github.com/HappyGuest/happyguestmx-web-utilities
+// Definitions by: HappyGuest <https://github.com/HappyGuest>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+declare module '@happyguestmx/web-utilities' {
```
Comment (andrewbranch) [inline types/happyguestmx__web-utilities/index.d.ts:7]: There’s no need for this wrapper.

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/happyguestmx__web-utilities/package.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "private": true,
+    "dependencies": {}
+}
```
Comment (andrewbranch) [inline types/happyguestmx__web-utilities/package.json:4]: Sorry I missed this, you can delete this file too.

**Review by andrewbranch — CHANGES_REQUESTED** [review]
Looks good after removing package.json

**Review by andrewbranch — APPROVED** [review]

**Comment by Alvarez-P** [comment]
Ready to merge

**Comment by andrewbranch** [comment]
@typescript-bot we’re waiting

**Comment by andrewbranch** [comment]
Oh, @Alvarez-P you can’t be ready to merge because you’re not @HappyGuest

**Comment by HappyGuest** [comment]
Ready to merge

**Comment by HappyGuest** [comment]
> Oh, @Alvarez-P you can’t be ready to merge because you’re not @HappyGuest

ty

**Pull request merged**